data/human/human missions.txt

```diff
@@ -5064,8 +5064,10 @@ mission "A dead man's path II"
 				`	"Now that we've come so far, sure."`
 					goto continue
 				`	"I'm sorry, I've got to be somewhere else."`
-			`	"Well, that's okay. We'll always appreciate the help you've given us in fulfilling our father's will! Oh, and here's your payment," Liam says, handing you credit chip of <payment>. "We wish you well through the stars!" Again, the siblings both thank you profusely for the trip, and you watch them leave Prime's spaceport with their heads up.`
-			payment 50000
+			`	"Well, that's okay. We'll always appreciate the help you've given us in fulfilling our father's will! Oh, and here's your payment," Liam says, handing you a credit chip for <payment>. "We wish you well through the stars!" Again, the siblings both thank you profusely for the trip, and you watch them leave Prime's spaceport with their heads up.`
+			action
+				payment 50000
+			decline
```

Comment:
double decline here and this doesnt work anyway, decline to be on a text element
Either put the action before the text or have this:
Empty text:
``
decline (indented ofc)